h/security/policy/top_level.py

```diff
@@ -34,9 +35,23 @@ def _load_identity(self, request):
 @RequestLocalCache()
 def get_subpolicy(request):
     """Return the subpolicy for TopLevelSecurityPolicy to delegate to for `request`."""
+    cookie = webob.cookies.SignedCookieProfile(
+        secret=request.registry.settings["h_auth_cookie_secret"],
+        salt="authsanity",
+        cookie_name="auth",
+        max_age=30 * 24 * 3600,  # 30 days
+        httponly=True,
+        secure=request.scheme == "https",
+    )
+    cookie = cookie.bind(request)
```

Comment:
Not sure I understand what this does, the docs say it binds it which I kinda expected:

https://docs.pylonsproject.org/projects/webob/en/stable/api/cookies.html#webob.cookies.CookieProfile.bind

Would binding multiple cookies to the same request have any effect?

Reply:
I don't think binding multiple cookies to the same request will be a problem, no. All `bind()` seems to do is return a new instance of `SignedCookieProfile` with `self.request` to to the given `request`: https://github.com/Pylons/webob/blob/5a67a56d32e3b7dd23b644b2c89736b799e50dc7/src/webob/cookies.py#L1084-L1102 All `self.request` appears to be used for (within the `cookie.py` file anyway) is reading cookies from the request (`self.request.cookies.get(cookie_name)`)